### PR TITLE
feat: support srt subtitle format

### PR DESCRIPTION
**Pull request by tsi**
This PR introduces a new `srtTextTracks` plugin that takes an SRT file, fetches it and convert it so it can be fed into `player.addRemoteTextTrack()`.

The code is roughly based on our styled-text-tracks & paced-transcript plugins and the conversion logic is inspired by https://github.com/silviapfeiffer/silviapfeiffer.github.io/blob/master/index.html#L150

### CODE REVIEW

File: docs/es-modules/subtitles-and-captions.html [hunk 1]
```diff
@@ -116,32 +116,32 @@ <h4 class="mt-4 mb-2">Karaoke player</h4>
       import 'cloudinary-video-player/cld-video-player.min.css';
       import 'cloudinary-video-player/playlist';
 
-      const player = videoPlayer('player', {
-        cloudName: 'demo'
+      const player = cloudinary.videoPlayer('player', {
+        cloud_name: 'prod'
```
Comment (jakub-roch) [inline docs/es-modules/subtitles-and-captions.html:120]: is it expected change?

Comment (tsi) [reply docs/es-modules/subtitles-and-captions.html]: The cloud change?  yes
Adding `cloudinary`? no, but It doesn't really matter

Comment (jakub-roch) [reply docs/es-modules/subtitles-and-captions.html]: I meant mostly about changing `cloudName` to `cloud_name`, isnt `cloudName` expected prop?

File: src/plugins/srt-text-tracks/srt-text-tracks.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    });
+  };
+
+  player.one('loadedmetadata', () => {
+    initSRT();
+  });
+}
+
+// SRT to WebVTT conversion functions
+const srt2webvtt = (data) => {
```
Comment (jakub-roch) [inline src/plugins/srt-text-tracks/srt-text-tracks.js:47]: what about using some ready lib like - https://www.npmjs.com/package/srt-parser-2 ?

**Review by jakub-roch — APPROVED** [review]
Looks good, just please see my comment about lib

File: src/plugins/srt-text-tracks/srt-text-tracks.js (hunk 2 above)
Comment (tsi) [reply src/plugins/srt-text-tracks/srt-text-tracks.js]: Good suggestion 💪

**Pull request merged**